### DISCUSSION TITLE
Trapdoors Consistency

### DISCUSSION THREAD

**Discussion by IdotIcom** [BedrockTweaks/Files#402] [Ideas]
Some trapdoors in Minecraft have handle and some don't. So I suggest adding.

a pack that removes the handle.
                 &
a pack that add handle.
——————————————————————————
![Screenshot_2024-11-18-08-33-21-17](https://github.com/user-attachments/assets/45850821-efa0-4c4b-b99d-233d5137fc3a)
Trapdoors that don't have handle. ↑↑↑
![Screenshot_2024-11-18-05-44-32-90](https://github.com/user-attachments/assets/5c88a821-2384-4f89-9186-5cdc42f045d2)
Trapdoors that have handle.↑↑↑

**Comment by SquatchHunter** [comment]
its not a hinge but a handle, and its more of a style choice
probably out of scope / not something we will do
of course can be considered for community packs later down the line
